net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -689,12 +666,8 @@ mwan3_delete_iface_rules()
 		return
 	fi
 
-	while [ -n "$($IP rule list | awk '$1 == "'$(($id+1000)):'"')" ]; do
-		$IP rule del pref $(($id+1000))
-	done
-
-	while [ -n "$($IP rule list | awk '$1 == "'$(($id+2000)):'"')" ]; do
-		$IP rule del pref $(($id+2000))
+	for i in $(ip rule list | awk '$1 % 1000 == '$id' && $1 > 1000 && $1 < 4000 {print substr($1,0,4)}'); do
```

Comment:
could you name the **i** to **rules** ?
On my vi you see **i** for insert and I sometimes don't know if this is a typo or not if I have a quick look on the code. Can you please change that.